data/zilliqa/assets/zil177dy26j6l4qj6wysug3j7cs97ejtaz2h06vfwa/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Metazoa",
+  "symbol": "ZOA",
+  "web": "https://zolar.io/",
+  "whitepaper": "https://docs.zolar.io/",
+  "links": {
+    "twitter": "https://twitter.com/bearmarketnft",
+    "discord": "https://discord.gg/zilswap",
+    "github": "https://github.com/switcheo/zilswap",
+    "facebook": "https://instagram.com/thebearmarketnft"
+  },
+  "holders": true,
+  "product": true,
```

Comment:
not much activity yet, remove

Reply:
the NFT mint just started today, holders will be able to play the Zolar game on zolar.io in the next few weeks. thanks